src/components/Article.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getArticle } from "../../api";
+import { useParams } from "react-router-dom";
+
+function Article() {
+  const [article, setArticle] = useState([]);
+
+  let { article_id } = useParams();
+
+  useEffect(() => {
+    getArticle(article_id)
+      .then((data) => {
+        console.log(data);
+        setArticle(data.article);
```

Comment:
Ideally we should be getting just an article back instead of an object with an article attached to it